card/mod/search/set/abstract/00_filter_helper.rb

```diff
@@ -29,11 +29,9 @@ def safe_sql_param key
 end
 
 def filter_keys_with_values
-  (filter_keys + advanced_filter_keys).map do |key|
+  filter_keys.map do |key|
```

Comment:
unless I misunderstand, `advanced_filter_keys` doesn't do anything any more; all the magic is now in `default_filter_option`